src/Sylius/Bundle/CoreBundle/Fixture/Factory/TranslatableExampleFactoryTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Fixture\Factory;
+
+use Sylius\Component\Resource\Model\TranslatableInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+trait TranslatableExampleFactoryTrait
+{
+    /** @var OptionsResolver */
+    private $optionsResolver;
+
+    abstract protected function createTranslation(TranslatableInterface $translatable, string $localeCode, array $options = []): void;
+
+    protected function configureTranslationsOptions(OptionsResolver $resolver): void
+    {
+        $resolver
+            ->setDefault('translations', [])
```

Comment:
I'm wondering if we shouldn't provide some way to define what translations we can expect to have here. Something similar to `createTranslation`. On the other hand, your proposal is identical to the solution we have already